src/joy_api.c

```diff
@@ -640,6 +640,11 @@ int joy_initialize_no_config(void *config, FILE *err_info, joy_init_t *data)
     JOY_API_ALLOC_CONTEXT(ctx_data, data->contexts)
     joy_num_contexts = data->contexts;
 
+    glb_config->num_pkts = DEFAULT_NUM_PKT_LEN;
+    if ((data->num_pkts > 0) && (data->num_pkts < MAX_NUM_PKT_LEN)) {
```

Comment:
I noticed that configure_t  has num_pkts is defined as uin16_t but joy_init_t has it uint8_t
We should probably keep them the same size since the both represent the pkts per flow.

Reply:
the max is 200 - so it really should be a uint8_t, but I am not changing that stuff at the moment - want minimal impact